### ISSUE TITLE
Toggle - a11y - light blue focus indicator is very light

### ISSUE THREAD

**Issue by arbydev** [carbon-design-system/carbon#572]
A light blue color of #b4d8fb is used on the Toggle circles when they have focus. Users with poor color contrast distinction may not even see it when it has focus. Could you try out some higher contrasting color options?

![image](https://user-images.githubusercontent.com/23506871/36226843-a58d0556-1194-11e8-8b41-73b8aa8e362b.png)

> Is this issue related to a specific component? 
Toggle

> What version of the Carbon Design System are you using? 
8.10.1

## Steps to reproduce the issue

1. Tab to the toggle to see the light blue circle.

## Add labels
a11y

**Comment by bsonefeld** [comment]
Hi @arbydev , thanks for pointing this out.  We did a big `:focus` state audit a few months ago where we created custom focus states for the components. We did discuss and do testing around making sure the focus outline was considered high contrast. 

Although we don't have plans to revisit our `:focus` states anytime soon within the next few months, it's something to keep in mind. 

CC: @tay-aitken
